web/components/templates/prompts/experiments/scoresTable.tsx

```diff
@@ -0,0 +1,197 @@
+          >
+            {`${changeInfo.change} (${changeInfo.percentageChange}%)`}
+          </span>
+        );
+      case "model":
+        return (
+          <span
+            className={clsx(
+              scores.dataset.scores.model === scores.hypothesis.scores.model
+                ? "bg-gray-50 text-gray-700 ring-gray-200"
+                : "bg-gray-50 text-gray-700 ring-gray-200",
+              "w-max items-center rounded-lg px-2 py-1 -my-1 text-xs font-medium ring-1 ring-inset"
+            )}
+          >
+            {scores.dataset.scores.model === scores.hypothesis.scores.model
+              ? "same"
+              : "changed"}
+          </span>
+        );
+      default:
+        return null;
+    }
+  };
+
+  const renderScoreValue = (value: any) => {
+    if (value instanceof Date) {
+      return value.toLocaleDateString();
+    }
+    if (typeof value === "string" && !isNaN(Date.parse(value))) {
+      return new Date(value).toLocaleDateString();
+    }
+
+    return value;
+  };
+  const renderTableRows = (scores: ExperimentScores) => {
+    if (!scores || !scores.dataset.scores) {
+      return null;
+    }
+
+    const experimentScoresAttributes = [
+      ...Object.keys(scores.dataset.scores).filter(
+        (key) => key !== "customScores"
+      ),
+    ];
+
+    return experimentScoresAttributes.map((field) => {
+      const calculateChange = (datasetCost: number, hypothesisCost: number) => {
+        const change = hypothesisCost - datasetCost;
+        const percentageChange = ((change / datasetCost) * 100).toFixed(2);
+        return {
+          change: parseFloat(change.toFixed(4)),
+          percentageChange,
+        };
+      };
+
+      return (
+        <TableRow key={field} className="w-full">
+          <TableCell className="h-full items-start border-r border-gray-300 max-w-xs">
+            <p className="text-black text-sm">{getScoreAttribute(field)}</p>
+          </TableCell>
+          <TableCell className="h-full border-l border-gray-300">
+            <p className="text-black text-sm">
+              {getScoreValue(scores.dataset.scores[field], field)}
+            </p>
+          </TableCell>
+          <TableCell className="h-full border-l border-gray-300">
+            <p className="text-black text-sm">
+              {getScoreValue(scores.hypothesis.scores[field], field)}
+            </p>
+          </TableCell>
+          <TableCell className="h-full border-l border-gray-300">
+            {renderComparisonCell(
+              field,
+              scores,
+              field === "cost" &&
+                calculateChange(
+                  scores.dataset.scores.cost as number,
+                  scores.hypothesis.scores.cost as number
+                )
+            )}
+          </TableCell>
+        </TableRow>
+      );
+    });
+  };
+  return (
+    <Table className="h-full w-full dark:border-gray-700 bg-white dark:bg-black p-4 border border-gray-300 rounded-lg">
```

Comment:
nit: try to use the SimpleTable component here. that way, we don't have to change this any time we want to make changes to the SimpleTable